tests/prototypes/prototype_test.py

```diff
@@ -1,40 +1,100 @@
 # SPDX-License-Identifier: BSD-3-Clause
 # Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
-from typing import Any, Generator
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Generator, Optional
 
 import pytest
 
-from beamlime.constructors import Factory
-from tests.benchmarks.runner import BenchmarkSession
-from tests.prototypes.parameters import EventRate, NumPixels, PrototypeParameters
-from tests.prototypes.prototype_mini import (
-    BenchmarkTargetName,
-    PrototypeBenchmarkRecipe,
-    mini_prototype_factory,
-)
+from beamlime.applications._parameters import EventRate, NumPixels, PrototypeParameters
```

Comment:
We should probably just remove most of these tests, since most of this will be handled completely differently. Otherwise we will waste endless time refactoring these tests again and again.

Reply:
I just removed all the tests for now.
I'll add tests later if needed.